service/src/main/java/org/kiwiproject/champagne/dao/BuildDao.java

```diff
@@ -15,66 +15,66 @@
 
 public interface BuildDao {
     
-    default List<Build> findPagedBuilds(int offset, int limit, String componentIdentifierFilter, String componentVersionFilter) {
+    default List<Build> findPagedBuilds(int offset, int limit, Long systemId, String componentIdentifierFilter, String componentVersionFilter) {
```

Comment:
Is this required? If so please create a new issue to change it to a primitive long